spec/features/user_cannot_view_admin_pages_spec.rb

```diff
@@ -1,3 +1,20 @@
-#create a user
-#log in as user
-#attempt to visit admin page
+require 'rails_helper'
+
+describe 'logged in user cannot view admin pages', type: :feature do
+  let(:user) { Fabricate(:user) }
+  let(:admin){ User.create(username: "admin", first_name: "admin", last_name: "admin", email: "admin@admin.com", password: "password", password_confirmation: "password", role: "admin")}
```

Comment:
Line is too long. [185/80]<br>Space missing to the left of {.<br>Space missing inside }.